GOVERNANCE.md

```diff
@@ -1,45 +1,117 @@
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. If this is not possible, Maintainers may call a vote. Many of the day-to-day project maintenance can be done by a [lazy consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) model. But the following items must be called to vote:
 
-Organizer Committee:  The Organizer Committee consists of organizers in charge of event or activity operations; in charge of the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions
+- Enforcing a Code of Conduct violation (super majority)
+- Adding or removing a maintainer/committer/SIG Leads (super majority)
+- Changing the governance rules in this document (super majority)
+- Licensing and intellectual property changes (including new logos, wordmarks) (simple majority)
+- Adding, archiving, or removing SIG/subproject (simple majority)
+- Other cases where consensus cannot be made (super majority)
 
-## Developer Group Structure
+For formal votes, a specific statement of what is being voted on should be added to the relevant github issue or PR, and a link to that issue or PR added to the maintainers meeting agenda document. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
 
-![developer group](/media/governance/developer_group.png)
+Voting must comply with the [Guiding Principles](/guiding-principles.md) and [CNCF Code of Conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md).
 
-Community groups include Special Interest Group (SIG) and Working Group (WG). The second part of the organizational structure illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion machanism.
+### Proposal process
 
-### Organization
+We use a [Request for Comments (RFC) process for any substantial changes to TiKV. This process involves an upfront design that will provide increased visibility to the community. If you're considering a PR that will bring in a new feature that may affect how TiKV is implemented, or may be a breaking change, then you should start with a RFC. We've got the process documented in [RFC repository](https://github.com/tikv/rfcs) and have a [template](https://github.com/tikv/rfcs/blob/master/template.md) for you to get started.
```

Comment:
```suggestion
We use a Request for Comments (RFC) process for any substantial changes to TiKV. This process involves some upfront design that will provide increased visibility to the community. If you're considering a PR that will bring in a new feature that may affect how TiKV is implemented, or may be a breaking change, then you should start with a RFC. The process is documented in [RFC repository](https://github.com/tikv/rfcs) and have a [template](https://github.com/tikv/rfcs/blob/master/template.md) for you to get started.
```

I think for all but the most obvious RFCs, the best way to start is by discussing with the relevant SIG in an informal way, then write the RFC, perhaps with help from someone in the SIG if there is interest. This avoids wasted work and disappointment

Reply:
Sounds good